modules/objdetect/test/test_boarddetection.cpp

```diff
@@ -318,4 +318,12 @@ TEST(CV_ArucoGenerateBoard, regression_1226) {
     });
 }
 
+TEST(CV_ArucoDictionary, extendDictionary) {
+    aruco::Dictionary base_dictionary = aruco::getPredefinedDictionary(aruco::DICT_4X4_250);
+    ASSERT_EQ(base_dictionary.bytesList.rows, 250);
+
+    aruco::Dictionary custom_dictionary = aruco::extendDictionary(150, 4, base_dictionary);
+    ASSERT_EQ(custom_dictionary.bytesList.rows, 150);
```

Comment:
It makes sense to check content of the new dictionary, as soon as it's subset of the original object.

Reply:
added check